src/common/labelValuePair/labelValuePair.scss

```diff
@@ -0,0 +1,12 @@
+@import 'spacings';
+@import 'colours';
+
+.vene-label-value-pair {
+  &__label {
+    font-weight: 500;
+  }
+
+  &__value {
+    margin-left: units(0.5);
+  }
+}
```

Comment:
Rename this file to `labelValuePair.module.scss`

Reply:
fixed